packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -37,12 +38,61 @@ export type Props = {|
 
     /** The `TooltipContent` element that will be rendered in the bubble. */
     children: React.Element<typeof TooltipContent>,
+
+    onActiveChanged: (active: boolean) => mixed,
+
+    /* A callback for updating the timeoutID refrence in tooltip  */
+    onTimeoutChanged: (timeoutID: ?TimeoutID) => mixed,
+
+    /* The timeoutID attached to the tooltip anchor */
+    anchorTimeoutID: ?TimeoutID,
+
     // TODO(somewhatabstract): Update react-docgen to support spread operators
     // (v3 beta introduces this)
     ...TooltipBubbleProps,
 |};
 
-export default class TooltipBubble extends React.Component<Props> {
+type State = {|
+    active: boolean,
+|};
+
+export default class TooltipBubble extends React.Component<Props, State> {
+    state = {
+        active: false,
+    };
+
+    componentDidMount() {
+        const bubbleNode = ReactDOM.findDOMNode(this);
+
+        if (!bubbleNode) {
+            return;
+        }
+
+        bubbleNode.addEventListener("mouseenter", this._handleMouseEnter);
```

Comment:
Maybe I don't have the full context here... I'd like to understand what is the reasoning behind using `addEventListener` vs using `onMouseEnter` as a prop attached to the main `View`.

Reply:
It's mainly because I was keeping the code consistent with tooltip-anchor.js, which uses `addEventListener`. However `onMouseEnter` should work perfectly fine too, I can change it if needed.

Reply:
Oh ok, I checked `tooltip-anchor.js` and for your case I think it's easier/better to use `onMouseEnter`. That way, you don't need to check if the element is already present in the DOM, and also don't need to add `removeEventListener` or any extra lifecycle events (`componentDidMount` and `componentWillUnmount` for the `removeEventListener` case)  b/c React handles that internally.